cloudslang-entities/src/main/java/io/cloudslang/lang/entities/WorkerGroupStatement.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.lang.entities;
+
+import io.cloudslang.lang.entities.bindings.ScriptFunction;
+import org.apache.commons.lang3.Validate;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.io.Serializable;
+import java.util.Set;
+
+public class WorkerGroupStatement implements Serializable {
+
+    private final String expression;
+    private Set<ScriptFunction> functionDependencies;
+    private Set<String> systemPropertyDependencies;
+
+    public WorkerGroupStatement(String expression, Set<ScriptFunction> functionDependencies,
```

Comment:
Personal preference: if all arguments don't fit on one line put one argument per line (first argument on the line with the constructor definition)